### ISSUE TITLE
t-stat in parenthesis

### ISSUE THREAD

**Issue by yonxie** [mwburke/stargazer#85]
I noticed you use standard error in the parenthesis below regression coeffcients. But many researchers use t-statistic which directly refect the significance of the coefficients. I find it doesn't request much work to add such flexibility. It would be great if you have it.

**Comment by toobaz** [comment]
You are right, this would be a nice addition (I actually thought it was supported already!)

A PR will be welcome.

**Comment by vlshields** [comment]
I have created a PR for this issue
